include/odp/api/spec/align.h

```diff
@@ -66,6 +66,15 @@ extern "C" {
  * Defines type/struct/variable to be page size aligned
  */
 
+/**
+ * @def ODP_CACHE_LINE_ROUNDUP
```

Comment:
Just a suggestion. Given that there is really a limited number of ways to implement this macro, why not move it to the helper?

Reply:
Yes, there are not many ways to implement it, but cache alignment is so commonly needed by applications that it's better to have this as a proper API.

Helpers are something that our test applications commonly re-use (e.g. protocol headers), but other applications could choose not to use (e.g. as they have their own definitions already).